### ISSUE TITLE
OE GUI (v0.5.5.4) crashes when record is pressed

### ISSUE THREAD

**Issue by SumedhNagrale** [open-ephys/plugin-GUI#508]
### GUI crashes with record node
Today, We updated the open ephys code to v0.5.5.4 for our lab repository at https://github.com/tne-lab/plugin-GUI/tree/low-latency
While doing the unit testing for the GUI, we found the OE GUI crashes whenever we press record with a record node in the signal chain as show below. 
![image](https://user-images.githubusercontent.com/51673422/165625382-217928d7-be18-4a6f-9302-a7be0fcc95d9.png)
This is also true when we simply press play with record node in the chain. 

### GUI works without record node
Further, we observed GUI works (I mean play) without Record Node in the signal chain as shown below
![image](https://user-images.githubusercontent.com/51673422/165630805-022d7de8-93c7-4500-a777-c2dc1531c181.png)

### Same issue with the code from https://github.com/open-ephys/plugin-GUI (master branch) 
To check _if this issue is caused by additional plugins we installed_ **or** _if it is caused due to the update we performed_. 
We downloaded the Open Ephys GUI code from https://github.com/open-ephys/plugin-GUI (master branch) and build the project in release mode using VS 2017 after the cmake -G "Visual Studio 15 2017 Win64" ..  
We found the same issue in the GUI (crashes with Record node and works without Record node), for reference here is the image
![image](https://user-images.githubusercontent.com/51673422/165629232-053e8564-bcaf-49fe-a0bf-9e13951ffb88.png)

Further, we repeated this process with VS 2019 and found the issue to be present in the GUI. 

### Debug Console 
If this is useful here is the debug console when GUI crashes
![image](https://user-images.githubusercontent.com/51673422/165632741-a81c0442-d7eb-48cc-a8e6-ed591a04242d.png)

Please, let me know if you need additional information about it and also, if I did something incorrectly above.

Thank you!
Sumedh

**Comment by anjaldoshi** [comment]
Hi Sumedh,

Thanks for reporting this. I was able to reproduce this and I've figure out a fix for it. We missed a message center event channel creation call while backporting a fix from our upcoming release (v0.6.0) which lead to this behavior. The fix is up on the `development` branch in this commit : 73805cb. It'll get merged to master today after we make sure everything looks good.

**Comment by anjaldoshi** [comment]
Fix has been merged to master. Updated v0.5.5.4 binaries are available to download now. 
Let us know if you encounter any further issues.

**Comment by SumedhNagrale** [comment]
Hi Anjal,

I am really looking forward to the v0.6.0 (especially with the JUCE update). 
Thank you for the fix!

**Comment by anjaldoshi** [comment]
Hi Sumedh,

We encountered another issue with v0.5.5.4 related to audio monitoring causing crashes and another one related to merger connections. I've pushed bug-fixes for both of them on the master branch and also updated the v0.5.5.4 binaries. Make sure you are on the latest code to avoid any issues related to this.

Regarding v0.6.0, we just did the final beta release of v0.6.0, and the next release of v0.6.0 is going to be the stable release. So, hopefully very soon :)
